### PR TITLE
 [62] Create & Style the Customer Signup Page using CSS

### PR DESCRIPTION
**Pull request by Marienoir**
## Task: ` Create & Style the Customer Signup Page using CSS`
- ` I created & Styled the Customer Signup Page using CSS`
Capture
Capture1
Capture3

### CODE REVIEW

**Comment by adaugochi** [comment]
Well done @Marienoir 

But, I think there is more you can do. Spaces are not evenly distributed in the `body` section of your work, i.e `left and right`. Also the same goes for the form card. I can't see your footer section.

Kindly fix codefactor issue on your PR

File: submissions/maryOkosun/html-forms/contact.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
 <head>
 	<meta charset="utf-8">
 	<title>PhoneHub</title>
+
```
Comment (adaugochi) [inline submissions/maryOkosun/html-forms/contact.html:6]: remove spaces

**Review by adaugochi — CHANGES_REQUESTED** [review]
remove spaces on image names

Also, send a screenshot on your new implementation

**Comment by Marienoir** [comment]
phone1
phone2
tablet1
tablet2
web1
web2

**Comment by adaugochi** [comment]
Well done @Marienoir 

We are making progress
Kindly let the text at the top to occupy available spaces (i.e left and right ) https://user-images.githubusercontent.com/40265625/97501287-4dab4180-1971-11eb-97ab-a5d05a52d837.PNG

**Comment by Marienoir** [comment]
phone3

**Review by adaugochi — CHANGES_REQUESTED** [review]
We want to try as much as possible to separate concerns. Kindly place all images inside the `images` folder that is in your `html` folder

**Review by adaugochi — CHANGES_REQUESTED** [review]
You have two `signup.html` file one inside the `css` folder and the other inside your `html-form`, Kindly remove the one in your `css` folder

**Review by adaugochi — APPROVED** [review]

**Pull request merged**